build.gradle

```diff
@@ -139,40 +137,6 @@ task('copyResources') {
     }
 }
 
-tasks.getByName('eclipseJdtApt') {
-    dependsOn gradle.includedBuilds*.task(':eclipseJdtApt')
-}
-tasks.getByName('eclipseJdt') {
-    dependsOn gradle.includedBuilds*.task(':eclipseJdt')
-}
-tasks.getByName('eclipseFactorypath') {
-    dependsOn gradle.includedBuilds*.task(':eclipseFactorypath')
-}
-eclipse {
-	// re-generate APT configuration on Gradle project synchronisation
-	//synchronizationTasks eclipseJdtApt, eclipseJdt, eclipseFactorypath
-
-	//jdt.apt {
-    //    genSrcDir = file('bin/generated/sources/annotationProcessor/java/main')
-    //    genTestSrcDir = file('bin/generated/sources/annotationProcessor/java/test')
-    //}
-
-    classpath.file {
-        //beforeMerged { classpath -> 
-        //    sourceSets.main.java.srcDirs += "bin/generated/sources/annotationProcessor/java/main"
-        //    sourceSets.test.java.srcDirs += "bin/generated/sources/annotationProcessor/java/test"
-        //}
-
-        //whenMerged { classpath ->
```

Comment:
This is what was supposed to hide warnings, but it doesn't seem to work with latest java plugin